data/korath/korath missions.txt

```diff
@@ -257,3 +257,588 @@ mission "Small Scale Delivery"
+					goto exile
+					to display
+						has "Korath [F05]: The Rainbow Feather: know"
+						not "Korath [F05]: The Rainbow Feather: exile"
+				`	"What is the nature of this fish trade?"`
+					goto freedom
+					to display
+						has "Korath [F05]: The Rainbow Feather: know"
+						not "Korath [F05]: The Rainbow Feather: freedom"
+
+			label harm
+			action
+				set "Korath [F05]: The Rainbow Feather: know"
+			`	"The Korath told me what you did: you brought a weapon to Far'en Lei, Captain." She looks deeply concerned. "That, alone, is terrible harm, and you should never do it again. They let you go because it is a planet of peace."`
+				goto check
+
+			label know
+			action
+				set "Korath [F05]: The Rainbow Feather: know"
+			`	"We are allies of the Korath. Naturally, if a human somehow obtains a jump drive and wanders into Korath space, they ask us about it. Landing on Far'en Lei with a weapon does terrible harm. They let you go because it is a planet of peace."`
+				goto check
+
+			label feather
+			action
+				set "Korath [F05]: The Rainbow Feather: feather"
+			`	She removes her necklace and cups the rainbow feather tenderly in her hands. "This was a gift from a Korath woman who is my... close friend. Korath keep memories in their skin and pass knowledge by eating another's memories. The stronger the memory, the more colorful the feathers. This feather has the memories and emotions that she identifies with me.`
+			`	"If she dies in a raid, and another Korath eats this, I'll have her back, in a way. Another Korath that feels for me as she does, and remembers our times together. All Korath know the consequence of a rainbow feather. This is how they pass all knowledge; not through schooling, but through skin.`
+			`	"Think about that a moment. Years of memories flow in as if they were yesterday. Powerful emotions overwhelm you, and you don't understand why, because they aren't your own. You love someone who doesn't love you back because they never met you. Knowledge you couldn't comprehend before is intuitive, but it came at a high price. You'd lose all sense of time and self - unimaginable psychological pain to us is simply a fact of life to a Korath.`
+			`	"That, Captain, is the very core of their being. They never developed diplomacy because they didn't need it: a few feathers or the skin of the dead will end a conflict. That is, unless their terrible weapons annihilate the corpses, or if the enemies aren't Korath at all. We do what we can to mediate their troubles with humans, but we aren't always around."`
+				goto check
+
+			label freedom
+			action
+				set "Korath [F05]: The Rainbow Feather: freedom"
+			`	"There are few life-supporting planets in the systems where the Korath are exiled. One is the violent planet you visited: Rabasot Pa'ah. The Korath discovered New Tortuga has a similar, though less violent, environment and evolved a fish-like species that would be an excellent food source.`
```

Comment:
```suggestion
			`	"There are few life-supporting planets in the systems where the Korath are exiled. One is the violent planet you visited: Rabasot Pa'ah. The Korath discovered New Tortuga has a similar, though less violent, environment and developed a fish-like species that would be an excellent food source.`
```
Presumably they used genetic modification, selective breeding, or just designed an organism from scratch? Rather than getting it to actually evolve, I mean.

Reply:
The species formed naturally, over millions of years of evolution on New Tortuga. It is so exotic and specific that even the Korath couldn't reproduce it.

Reply:
Ah, I get it. I was reading it as the korath who evolved it. 

How about;

`The Korath discovered New Tortuga has a similar, though less violent, environment which evolved a fish-like species that would be an excellent food source.`

I think this makes it clearer that it’s the environment of New Tortuga which caused the fish to evolve, rather than the Korath.

Reply:
I reworked the paragraph and gave the fish species a name. How does it look now?

Reply:
Looks good to me!